Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -2375,15 +2376,26 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
 {
     Vector<PageIdentifierTuple> result;
 
-    for (auto& tab : openTabs()) {
-        auto window = tab->window();
-        for (WKWebView *webView in tab->webViews())
-            result.append({ webView._page->webPageID(), tab->identifier(), window ? std::optional(window->identifier()) : std::nullopt });
+    for (auto entry : m_extensionPageTabMap) {
```

Comment:
should `WebExtensionContext::getTab` also iterate through `m_extensionPageTabMap` instead of `openTabs())`?

Reply:
Yeah I think that would be good!